python-packages/demos/src/index.rst

```diff
@@ -0,0 +1,68 @@
+.. source for the sphinx-generated build/docs/web/index.html
```

Comment:
I think this content belongs in the `sra_client` module docstring, and the sphinx generation should happen there too.